java/src/main/java/com/genexus/GxImageUtil.java

```diff
@@ -22,7 +23,13 @@ private static InputStream getInputStream(String filePathOrUrl) throws IOExcepti
 
 	private static BufferedImage createBufferedImageFromURI(String filePathOrUrl) throws IOException
 	{
-		try (InputStream is = getGXFile(filePathOrUrl).getStream()) {
+		InputStream is;
```

Comment:
Input stream should be used using the 'try-with-resources' in order to be sure that all resources are closed properly (prevent memory leaks).
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html